reana-ui/src/pages/workflowList/WorkflowList.js

```diff
@@ -92,8 +93,6 @@ function Workflows() {
               interval.current = null;
```

Comment:
I guess you can also remove the `defaultActivePage` here (4 lines above).

Reply:
Yep, I tried that, it seems that `semantic-ui-react` does not set this value by default, so removing it results in an error:
```Received NaN for the `value` attribute. If this is expected, cast the value to a string.```.
So it ether have to be `defaultActivePage` or `activePage` prop set for this to work correctly.

Reply:
Ah ok, I see 👍